core/tests/protractor/embedding.js

```diff
@@ -62,8 +62,8 @@ describe('Embedding', function() {
       'Exploration completed'
     ];
 
-    users.createUser('embedder@example.com', 'Embedder');
-    users.login('embedder@example.com', true);
+    users.createUser('embedder@Embedding.com', 'Embedder');
```

Comment:
Should I name it Embedder or
Embedder_embedding
or
user1_embedding
or
alice_embedding